source/features/conflict-marker.css

```diff
@@ -0,0 +1,5 @@
+.rgh-conflict-marker {
+	margin: 0;
+	margin-right: 4px;
+	color: #cb2431;
```

Comment:
Change this a yellow shade, red looks the same as CI errors.

Reply:
Perhaps it should just be gray. 

Also all of these values should be replaced with native classes, like `m-0` (I think) `text-gray` and `mr-2`